install/install-by-msi.md

```diff
@@ -74,7 +74,7 @@ server process. Please execute `Td-agent Command Prompt` again but with
 administrative privilege, and type the two commands below.
 
 ```
-> fluentd --reg-winsvc i
+> fluentd --reg-winsvc i --reg-winsvc-auto-start --reg-winsvc-delay-start
 > fluentd --reg-winsvc-fluentdopt '-c C:/opt/td-agent/etc/td-agent/td-agent.conf -o C:/opt/td-agent/td-agent.log'
 ```
 
```

Comment:
I think that example should be minimal.
`--reg-winsvc-auto-start` and `--reg-winsvc-delay-start` should be in **NOTE:**.

---

**NOTE:** Automatic td-agent service requires the additional commandline parameters:

```
>  fluentd --reg-winsvc i --reg-winsvc-auto-start --reg-winsvc-delay-start
```

And then, register fluentd opt with `--reg-winsvc-fluentdopt` option like as the above example.

Reply:
Done. Is this what you had in mind?